README.md

```diff
@@ -100,6 +100,8 @@ if let j: AnyObject = json {
 
 For more information, see the [Documentation](Documentation/)
 
+Please note: the above example requires an additional, external module named [Curry](https://github.com/thoughtbot/Curry) to automatically curry `User.init`.
```

Comment:
True, "automatic" was a poor choice of wording. I'll make those changes once I'm back at my computer :+1:

> On Oct 20, 2015, at 4:01 PM, Tony DiPasquale <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -100,6 +100,8 @@ if let j: AnyObject = json {
> >  
> >  For more information, see the [Documentation](Documentation/)
> >  
> > +Please note: the above example requires an additional, external module named [Curry](https://github.com/thoughtbot/Curry) to automatically curry `User.init`.
> also it's not automatic so we should say "to use the curry function to curry User.init." or something like that
> 
> —
> Reply to this email directly or view it on GitHub.
>